mathjax3-ts/output/chtml/Wrappers/TextNode.ts

```diff
@@ -51,13 +51,24 @@ export class CHTMLTextNode extends CHTMLWrapper {
      * @override
      */
     public computeBBox() {
-        //
-        // Fake a bbox for now (eventually, this will get looked up in the font table)
-        //
-        this.bbox.w = .5;
-        this.bbox.h = .8;
-        this.bbox.d = .2;
-        return this.bbox;
+        const variant = this.parent.variant;
+        let bbox = this.bbox;
+        if (variant === '-explicitFont') {
+            // FIXME:  measure this using DOM, if possible
```

Comment:
This is the location where the code to measure the characters using the DOM will go (or be linked in). The comment is there to indicate that there is more coming in the future.  If it would make you happier, I can add a call to something like `this.getDomBBox()` at this point, and make an empty `getDomBBox()` method.

Reply:
Not necessary.